scripts/branch-deployment.sh

```diff
@@ -53,8 +67,10 @@ for x in $ALL_FOLDERS_DEPTH_1; do
     fi
 done)
 
-git config --global user.email "raphael.lueckl@axa.ch"
-git config --global user.name "Donald Duck"
+# Outdated branches are removed at this point, and the current branch is up to
+# date. Ready to publish!
+
+[[ $1 != "--my-credentials" ]] && git config --global user.email "raphael.lueckl@axa.ch" && git config --global user.name "Donald Duck"
```

Comment:
If i run that code locally my global git config is overwritten by your email and "Donald Duck"? Isnt it possible to give each git command those credential and leave out the global config?

Reply:
I think that is not possible. What I could implement is, that '--my-credentials' would be the default. But to make this work, I would have to change the existing pipeline-configs.

If you run `npm run deploy-branch`, it will always take your own credentials.

Reply:
For me changing the git globals to your email is not such a good idea. Any commits after that is with your email. You could only take "my-credentials" - and if its not given give an error

Reply:
Ok, if you want that, I can implement it like that. But this takes some extra time.